.g8/checkboxPage/app/viewmodels/checkAnswers/$className$Summary.scala

```diff
@@ -1,4 +1,4 @@
-package viewmodels.checkAnswers
+package viewmodels.detailsPages
```

Comment:
For this and the other scaffold pages below, you will also need to change the location of the file to be the same as the package name

Reply:
This was an adjustment to 1 file because it existed in 2 different contexts.  The remaining files are where they belong.

Reply:
Not 100% convinced why they should be different? Can discuss tomorrow

Reply:
updated name and location to match what is occurring